src/main/java/com/j256/ormlite/db/H2DatabaseType.java

```diff
@@ -52,6 +56,20 @@ public void appendLimitValue(StringBuilder sb, long limit, Long offset) {
 		sb.append(limit).append(' ');
 	}
 
+	@Override
+	public void appendOffsetTimeType(StringBuilder sb, FieldType fieldType, int fieldWidth) {
+		sb.append("TIMESTAMP WITH TIME ZONE");
+	}
+
+	@Override
+	public FieldConverter getFieldConverter(DataPersister dataPersister, FieldType fieldType) {
+		// H2 doesn't support TIME WITH TIME ZONE
+		if (dataPersister.getSqlType() == SqlType.OFFSET_TIME)
+			return DataType.OFFSET_TIME_SQL.getDataPersister();
```

Comment:
Following the change in -core, this should now be `OFFSET_TIME_COMPAT`.

Reply:
Fixed